tests/suite/zqd/services.sh

```diff
@@ -28,7 +33,7 @@ export AWS_ACCESS_KEY_ID=minioadmin
 export AWS_SECRET_ACCESS_KEY=minioadmin
 export AWS_S3_ENDPOINT=http://localhost:$(cat $portdir/minio)
 
-zqd listen -l=localhost:0 -portfile="$portdir/zqd" -datadir=./zqdroot &
+zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data=$zqdroot 2> zqd.error &
```

Comment:
$zqdroot might contain spaces.
```suggestion
zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" 2> zqd.error &
```
And I'm still in favor of changing the zqd logging level to Warn so we can drop `2> zqd.error`.

Reply:
The best way to do this is to have a `-loglevel` flag for zqd which we don't have. I'm going to do this in a follow up pr.